### PR TITLE
[esnext] split “global” into two subtests

### PR DESCRIPTION
**Pull request by ljharb**
Per #990.

### CODE REVIEW

**Review by chicoxyzzy — DISMISSED** [review]

File: data-esnext.js [hunk 1]
```diff
@@ -827,16 +827,42 @@ exports.tests = [
...
+      node7: true,
+    }
+  }, {
+    name: '"global" global property has correct property descriptor',
+    exec: function(){/*
+      var actualGlobal = Function('return this')();
+      if (typeof global !== 'object') { return false; }
+      if (!('global' in actualGlobal)) { return false; }
+      if (Object.prototype.propertyIsEnumerable.call(actualGlobal, 'global')) { return false; }
+      if (typeof Object.getOwnPropertyDescriptor !== 'function) { return true; } // ES3
```
Comment (chicoxyzzy) [inline data-esnext.js]: Can you explain this line please?

Comment (chicoxyzzy) [inline data-esnext.js]: got it

**Review by chicoxyzzy — APPROVED** [review]
OK I got it. LGTM

**Comment by chicoxyzzy** [comment]
just noticed we have no `node6`, there are [`node64` and `node65`](https://github.com/kangax/compat-table/blob/gh-pages/environments.json#L1410-L1439)

screen shot 2016-12-24 at 00 19 48

**Comment by ljharb** [comment]
whoops, k, will fix

**Comment by ljharb** [comment]
data-es6 is full of `node6` keys, perhaps it got lost in the transition to `environments.json`

**Comment by chicoxyzzy** [comment]
yes I think so... So It's probably better to rename `node64` to `node6`

**Comment by evilpie** [comment]
@ljharb There is a small syntax error at `Object.getOwnPropertyDescriptor !== 'function)`, note the missing `'` after `function`.

**Comment by ljharb** [comment]
@evilpie thanks, fixed

@chicoxyzzy probably true, but i didn't want to change the environments.json casually

**Comment by chicoxyzzy** [comment]
I am free now so I can fix all the node6/node64 collisions

**Pull request merged**